File: passed/1.1.0-test.cpp
/*
ID:   andy1li
PROG: test
LANG: C++         
*/

#include <fstream>
#include <iostream>
#include <string>

using namespace std;

int main() {
    ifstream fin  ("test.in");
    ofstream fout ("test.out");

    int a, b;
    fin >> a >> b;
    fout << a+b << endl;

    return 0;
}

File: Training/1.1 Dabble/1.1.0-test.py
"""
ID:   andy1li
LANG: PYTHON3
TASK: test
"""

def print_line(line): print(line); f_out.write( str(line) + '\n' )

task = 'test'
with open(task+'.in') as f_in, open(task+'.out', 'w') as f_out:
    input = f_in.readline

    a, b = map(int, input().split())
    print_line(a + b)